packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,204 +24,262 @@ _ref:
+            - `type: string`: __Required__ - This is the block type identifier and defines what block to used.
+            - `properties: object`: All the settings passed to a block component.
+            - `blocks: array`: An array of blocks to render within this block.
 
-            The config object has the following properties:
+            Find the more detailed block schema [here](/blocks).
 
-            - `basePath: string`: Set the base path to serve the Lowdefy application from. This will route all pages under `https://example.com/<base-path>/<page-id>` instead of the default `https://example.com/<page-id>`. The basePath value must start with "/".
-            - `homePageId: string`: The pageId of the page that should be loaded when a user loads the app without a pageId in the url route. This is the page that is loaded when you navigate to `yourdomain.com`.
-
-      - id: alert1
+      - id: yaml_json_files_alert
         type: Alert
         properties:
-          type: warning
+          type: info
           showIcon: false
-          message: Init page is an experimental feature, that may disappear in future releases as well as the flag itself can be changed. Use at your own risk.
+          message: |
+            <h3> YAML file extensions </h3>
+            <p> Both files with the <code>.yaml</code> and <code>.yml</code> file extensions are supported as YAML files. </p>
+            <h3> JSON instead of YAML </h3>
+            <p> Since you can reference JSON files, you can build your app using JSON instead of YAML files. The <code>lowdefy.yaml</code> file needs to be a YAML file, but all other configuration can be in referenced JSON files. It also makes sense to use JSON instead of YAML if you are generating configuration using code. </p>
 
-      - id: md2
+      - id: md3
         type: MarkdownWithCode
         properties:
           content: |
-            # Global
+            ## Config
 
-            Any data that you wish to use in your app can be stored in the __global__ object, and accessed using the [`_global`](/_global) operator. This is a good place to store data or configuration that is used throughout the app, for example the url of a logo or configuration of a page, since then these are only written once, and can be updated easily.
+            The config object has the following properties:
 
-            The global object can also be modified using the [`SetGlobal`](/SetGlobal) action.
+            - `basePath: string`: Set the base path to serve the Lowdefy application from. This will route all pages under `https://example.com/<base-path>/<page-id>` instead of the default `https://example.com/<page-id>`. The basePath value must start with "/".
+            - `homePageId: string`: The pageId of the page that should be loaded when a user loads the app without a pageId in the url route. This is the page that is loaded when you navigate to `yourdomain.com`.
```

Comment:
This sentence does not read well.. 
Maybe:
The id of the page that is loaded when you navigate to your home route, for example,`yourdomain.com`

Reply:
Change has been made in #1314.